runtime/doc/builtin.txt

```diff
@@ -1958,6 +1960,12 @@ cursor({list})
 		Can also be used as a |method|: >
 			GetCursorPos()->cursor()
 
+cursor_off()						*cursor_off()*
+		Disable cursor like |:sleep!|.
+
```

Comment:
i may suggest:
1. mentioned it do would recover cursor in 1 second anyhow, tho it may repeatly to be 'off'.
2. when 'off', perhaps the cmdline should indicate the status msg about the cursor was off.

Reply:
Disable is a bit vague? I guess what is meant it *hide* cursor?